api/src/test/scala/uk/ac/wellcome/platform/api/models/DisplayItemTest.scala

```diff
@@ -45,16 +45,19 @@ class DisplayItemTest extends FunSpec with Matchers {
 
   it("should read an Item as a DisplayItem correctly") {
 
-    val internalItem = Item(
-      identifiers = List(identifier),
-      locations = List(location)
-    )
+    val item = Item(
+        Some("foo"),
```

Comment:
It would be nice if this was either a keyword argument (`Item(id = Some("foo") ...`) or the `"foo"` was more clearly an identifier. Or both.